private/dump_new_fish_data.py

```diff
@@ -727,12 +727,12 @@ def get_location_key(spot):
             'bigfish': fish.item.rarity >= 2,
             'quest': len(fish.quest) > 0,
             # 'shop': len(fish.shop) > 0,
-            'satisfaction': fish.satisfaction is not None,
+            'satisfaction': fish.satisfaction['Collectability{High}'] if fish.satisfaction else None,
             'craft': len(fish.craft) > 0,
             'gc': fish.gc is not None,
             'leve': len(fish.leve) > 0,
-            'scrip': fish.scrip is not None,
-            'reduce': fish.reduce,
+            'scrip': fish.scrip['CollectablesShopRefine']['LowCollectability'] if fish.scrip else None,
+            'reduce': 1 if fish.reduce else None,
```

Comment:
I think this, `reduce`, was fine being true/false.

Reply:
You're right, the 1/None is pointless here